README.md

```diff
@@ -30,7 +30,7 @@ Users can call a high level `generate()` method, or run each iteration of the mo
 
 ## Coming very soon
 
-* Support for the Mistral and Whisper model architectures
+* Support for the Whisper model architectures
```

Comment:
We do have Whisper support in there now, but it just doesn't handle multiple audio chunks